README.org

```diff
@@ -209,24 +209,24 @@ You can find an exhaustive overview of all functions, their keybinds and functio
 
       (treemacs-follow-mode t)
       (treemacs-filewatch-mode t))
-    :bind
-    (:map global-map
-          ([f8]         . treemacs-toggle)
-          ("M-0"        . treemacs-select-window)
-          ("C-c 1"      . treemacs-delete-other-windows)
-          ("M-m ft"     . treemacs-toggle)
-          ("M-m fT"     . treemacs)
-          ("M-m fB"     . treemacs-bookmark)
-          ("M-m f C-t"  . treemacs-find-file)
-          ("M-m f M-t") . treemacs-find-tag))
+
+    (bind-keys ([f8]         . treemacs-toggle)
+               ("M-0"        . treemacs-select-window)
+               ("C-c 1"      . treemacs-delete-other-windows))
   (use-package treemacs-projectile
     :defer t
     :ensure t
     :config
     (setq treemacs-header-function #'treemacs-projectile-create-header)
-    :bind (:map global-map
-                ("M-m fP" . treemacs-projectile)
-                ("M-m fp" . treemacs-projectile-toggle)))
+    (bind-keys :prefix-map treemacs-prefix-map
+               :prefix "M-m"
+               ("f t"     . treemacs-toggle)
```

Comment:
Treemacs keybinds should stay with treemacs. These snippets are pick-and-choose and not everyone will care about projectile.